### DISCUSSION TITLE
Is the official docker-compose file outdated? budibase.docker.scarf.sh/budibase/apps or budibase/apps?

### DISCUSSION THREAD

**Discussion by bayeslearnerold** [Budibase/budibase#10388] [Help] [answered]
In the documentation, 
It pulls budibase.docker.scarf.sh/budibase/apps etc,  is this correct?  

Thanks.

**Comment by ConorWebb96** [comment, marked as answer]
Hey @bayeslearner,

I don't believe this is outdated, looking at the CLI and general codebase's docker-compose files they all have this included in them; `budibase.docker.scarf.sh/budibase/apps`

I hope this helps!

**Comment by bayeslearnerold** [comment]
If you use that instead of "budibase/apps" etc, you only get budibase version up to 2.4.
